src/components/Public/PortalSubmission.js

```diff
@@ -23,26 +23,125 @@ const PortalSubmission = () => {
     const [videoName, setVideoName] = useState("");
 
     const [submission, setSubmission] = useState(false);
+    const [validEmail, setValidEmail] = useState(false);
 
     // [Title to be shown, id of title for database]
     const requiredFields = [
-        ["First name", "firstName"],
-        ["Last name", "lastName"],
-        ["BU Email address", "email"],
-        ["Phone number", "phone"],
-        ["Expected year of graduation", "gradYear"],
-        ["Major", "major"]
+        {
+            label: 'First name',
+            name: 'firstName',
+            type: 'text'
+        },
+        {
+            label: 'Last name',
+            name: 'lastName',
+            type: 'text'
+        },
+        {
+            label: 'Major',
+            name: 'major',
+            type: 'text'
+        },
+        {
+            label: 'Expected graduation date',
+            name: 'gradYear',
+            type: 'date'
+        },
+        // {
+        //     label: 'Email address',
+        //     name: 'email',
+        //     type: 'email'
+        // },
+        // {
+        //     label: 'Phone number',
```

Comment:
Can you put the phone number back in the required fields? I don't think validation for phone number matters too much so just put the fields in but don't worry about regex.